client/my-sites/domains/domain-management/components/point-to-wpcom-dialog.tsx

```diff
@@ -0,0 +1,42 @@
+import { useI18n } from '@wordpress/react-i18n';
+import AcceptDialog from 'calypso/lib/accept/dialog'; // eslint-disable-line no-restricted-imports
+
+type PointToWpcomDialogProps = {
+	visible: boolean;
+	onClose: ( accepted: boolean ) => void;
+};
+
+export const PointToWpcomDialog = ( { visible, onClose }: PointToWpcomDialogProps ) => {
+	const { __ } = useI18n();
+
+	const renderContent = () => {
+		return (
+			<>
+				<p>{ __( 'When you point your domain to WordPress.com, we will:' ) }</p>
+				<ul>
+					<li>{ __( 'Change your name servers to use the WordPress.com defaults,' ) }</li>
+					<li>{ __( 'Delete any A records, and' ) }</li>
+					<li>{ __( "Delete any 'www' CNAME records" ) }</li>
```

Comment:
When I read this, I get the feeling that the domain will end up with no `A` records and no `www` `CNAME` records 😅 what do you think of something like "Reset to default A records" and "Set the default 'www' CNAME record"?